academic-services/knowledge-exploration-service/tutorial-entity-linking.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Entity linking
+description: Step by step tutorial to link private paper data with MAKES paper entities
+ms.topic: tutorial
+ms.date: 10/15/2020
+---
+
+# Entity Linking
+
+This tutorial illustrates how to link private paper data with MAKES paper entities. In this tutorial, we will show you how to link our sample library papers with MAKES paper entities using MAKES Interpret API.
+
+## Prerequisites
+
+- [Microsoft Academic Knowledge Service (MAKES) subscription](get-started-setup-provisioning.md)
+- [Powershell 7](https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-windows?view=powershell-7)
+
+## Select entity attributes to enrich our data
+
+We start by determining which MAKES entity type do we want to link our data against and what attributes can help enrich our private data. See [MAKES Entity Schema](reference-makes-api-entity-schema.md) to explore the different types and entities and attributes available in MAKES' default index.
+
+In this tutorial, the [sample library data](samplePrivateLibraryData.json) is bare bones and only contains paper title and on-prem full text link. We want to add more paper information and enable semantic/keyword search by linking against MAKES' paper entity.
```

Comment:
>We want to add more paper information and enable semantic/keyword search by linking against MAKES' paper entity. [](start = 147, length = 112)

"The goal of the tutorial is to augment the minimal library paper records with additional metadata from MAKES, enabling a robust semantic/keyword search experience."